src/utils.js

```diff
@@ -119,55 +119,109 @@ module.exports = class Utils {
   static selectFiles(config = {}) {
     return new Promise((resolve, reject) => {
       /**
-       * Create a new INPUT element
-       * @type {HTMLElement}
+       * Check cached INPUT element
+       * If inputElement is null, create INPUT element cache
        */
-      let inputElement = document.createElement('INPUT');
+      if (inputElement === null) {
+        /**
+         * Create a new INPUT element
+         * @type {HTMLElement}
+         */
+        inputElement = document.createElement('INPUT');
 
-      /**
-       * Set a 'FILE' type for this input element
-       * @type {string}
-       */
-      inputElement.type = 'file';
+        /**
+         * Set a 'FILE' type for this input element
+         * @type {string}
+         */
+        inputElement.type = 'file';
 
-      if (config.multiple) {
-        inputElement.setAttribute('multiple', 'multiple');
-      }
+        /**
+         * Do not show element
+         */
+        inputElement.style.display = 'none';
 
-      if (config.accept) {
-        inputElement.setAttribute('accept', config.accept);
-      }
+        /**
+         * Append element to the body
+         * Fix using module on mobile devices
+         */
+        document.body.appendChild(inputElement);
 
-      /**
-       * Do not show element
-       */
-      inputElement.style.display = 'none';
+        /**
+         * Create event handler for resolving files
+         * @param {Event} event
+         */
+        const resolveFiles = event => {
+          /**
+           * Get files from input field
+           */
+          const files = event.target.files;
 
-      /**
-       * Append element to the body
-       * Fix using module on mobile devices
-       */
-      document.body.appendChild(inputElement);
+          /**
+           * Return ready to be uploaded files array
+           */
+          resolve(files);
+
+          /**
+           * Remove event handler for preventing memory leak
+           */
+          inputElement.removeEventListener('change', resolveFiles);
+
+          /**
+           * Remove element from a DOM
+           */
+          document.body.removeChild(inputElement);
+
+          /**
+           * Remove input element
```

Comment:
but it is not removing of the element

Reply:
Hmm, Could you recommend a better comment?
I think... "Clear input element for preventing memory leak" is better.